core/src/main/java/com/novoda/noplayer/model/AudioTracks.java

```diff
@@ -0,0 +1,78 @@
+package com.novoda.noplayer.model;
+
+import com.novoda.noplayer.internal.exoplayer.mediasource.AudioTrackType;
+import com.novoda.utils.Optional;
+
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+public class AudioTracks implements Iterable<PlayerAudioTrack> {
+
+    private final List<PlayerAudioTrack> audioTracks;
+
+    public static AudioTracks from(List<PlayerAudioTrack> audioTracks) {
+        return new AudioTracks(Collections.unmodifiableList(audioTracks));
+    }
+
+    private AudioTracks(List<PlayerAudioTrack> audioTracks) {
+        this.audioTracks = audioTracks;
+    }
+
+    public boolean containsPlayerAudioTrackWith(AudioTrackType trackType) {
```

Comment:
this _is_ the sort of thing you could add to a first class collection but not sure if it'd be better to wait to see if there's a need across projects? Easy to add later but adding it now would mean we're stuck and breaking change to remove it later.

`containsTrackWith(AudioTrackType)` for brevity?